tests/lcs/core/engine/behavior.livecodescript

```diff
@@ -148,4 +148,26 @@ on TestBehaviorHandlerResolveOrder
    dispatch function "returnValue" to tTarget
    TestAssert "behavior handles passed dispatched function", \ 
       the result is false
-end TestBehaviorHandlerResolveOrder
\ No newline at end of file
+end TestBehaviorHandlerResolveOrder
+
+on TestChangeBehaviorWhileExecuting
+   create stack "test"
+   set the defaultStack to "test"
+   create button "child"
+   set the script of button "child" to "on Foo;Bar;end Foo"
+   create button "parent"
+   set the script of button "parent" to "on Bar;Baz;end Bar"
+   set the behavior of button "child" to the long id of button "parent"
```

Comment:
Is this line meant to be here? The first test seems to be saying the behavior starts off empty

Reply:
Ah good catch!